mobile/lib/src/views/setup_action_page.dart

```diff
@@ -43,7 +45,10 @@ class _SetupActionPageState extends State<SetupActionPage> {
       onChanged: (service) {
         setState(() {
           serviceState = service;
-          // TODO call api to get available actions
+          availableActions = [];
+          GetIt.I<AerisAPI>().getActionsFor(serviceState!, widget.action).then((actions) => setState(() {
```

Comment:
You should probably not run a setState in a setState, but I think the then is the right way